community/detectors/apache_http_server_cve_2021_41773/src/main/java/com/google/tsunami/plugins/detectors/rce/cve_2021_25646/ApacheHttpServerVulnDetector.java

```diff
@@ -0,0 +1,115 @@
+package com.google.tsunami.plugins.detectors.rce.cve_2021_25646;
```

Comment:
[Google Java Code Style](https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names): Package names are all lowercase, with consecutive words simply concatenated together (no underscores). For example, `com.example.deepspace`, not `com.example.deepSpace` or `com.example.deep_space`.

Reply:
done.